alf/algorithms/entropy_target_algorithm.py

```diff
@@ -122,7 +122,7 @@ def train_step(self, distribution):
                     alpha_loss=alpha_loss, entropy_loss=entropy_loss)))
 
     def calc_loss(self, training_info):
-        if self._debug_summaries and common.should_record_summaries():
```

Comment:
Why was this condition removed?

Reply:
there's only one simple summary `tf.summary.scalar` under this condition , and  it checks `should_record_summaries` in its implementation